builder/build.go

```diff
@@ -97,11 +98,11 @@ func CopyFiles(src string, destination string, recursive bool) {
 
 		if file.IsDir() == false {
 
-			cp(src+"/"+file.Name(), destination+file.Name())
+			cp(filepath.Join(src, file.Name()), filepath.Join(destination, file.Name()))
```

Comment:
I don't think this change is relevant to the Dockerfile language template. Can you comment please?

Reply:
It's not relevant, I can make a PR regrouping changes related to coding style, tell me if you prefer that way :-)